cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/GraphCommand.java

```diff
@@ -110,7 +110,7 @@ public void execute() {
 
         TaskExecutor taskExecutor = new TaskExecutor.TaskBuilder()
                 .addTask(new CleanTargetDirTask(true, false), isSingleFileProject())
-                .addTask(new RunBallerinaPreBuildToolsTask(outStream))
+                .addTask(new RunBallerinaPreBuildToolsTask(outStream), isSingleFileProject())
```

Comment:
Imo, Normaly we use `is` prefix into function. We can avoid using `is` prefix into the variable.
wdyt?